### PR TITLE
browser(firefox): roll Firefox stable to M98

### PR DESCRIPTION
**Pull request by aslushnikov**
The release of Firefox 98 is scheduled for Mar 8, 2022.

### CODE REVIEW

File: browser_patches/firefox/preferences/playwright.cfg [hunk 1]
```diff
@@ -14,6 +14,12 @@ pref("browser.tabs.remote.useCrossOriginOpenerPolicy", false);
 
 pref("browser.tabs.remote.separatePrivilegedMozillaWebContentProcess", false);
 
+pref("pdfjs.disabled", true);
```
Comment (mxschmitt) [inline browser_patches/firefox/preferences/playwright.cfg:17]: I'm curious, why did do we disable it?

Comment (aslushnikov) [reply browser_patches/firefox/preferences/playwright.cfg:17]: This way navigating to PDF triggers download instead of opening the inline plugin.

Comment (mxschmitt) [reply browser_patches/firefox/preferences/playwright.cfg:17]: sure, but did it broke tests or are we just randomly adding it?

Comment (aslushnikov) [reply browser_patches/firefox/preferences/playwright.cfg:17]: it did!

**Pull request merged**